wagtail/contrib/search_promotions/templates/wagtailsearchpromotions/add.html

```diff
@@ -3,7 +3,7 @@
 {% block titletag %}{% trans "Add search promotion" %}{% endblock %}
 {% block content %}
```

Comment:
We should try to simplify the template by extending the generic templates (same with the edit view as well)